examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,174 @@
+ 1. It will create a 3 node cluster with 1 writer and 2 reader endpoint spread across 3 Azs by default.
+ 2. It will also create a proxy to connect to the aurora cluster.
+ 3. The management of database credential is done through the secret manager.
+ 4. The security group has 2 rules , one to allow the app to connect to the proxy and one within the security group
+     where the proxy can connect to the database (Make sure you have provided the correct the CIDR at the claim which is the allow CIDR on the security group from app.)
+ 5. The Aurora DB has been configured with logging and monitoring.
+
+### Steps to check the connectivity from a pod 
+Create an  policy to allow the pod to connect to the Aurora Database
+  
+```shell
+aws iam create-policy \
+--policy-name rdsproxy-access \
+--policy-document file://rdsproxy-access.json
+```
+Create a service account passing the above policy arn
+
+```shell
+POLICY_ARN=$(aws iam list-policies --query 'Policies[?PolicyName==`rdsproxy-access`].Arn' --output text)
+CLUSTER_NAME=$(aws eks list-clusters --query 'clusters[]' --output text | grep <YOUR-CLUSTER-NAME>)
+
+eksctl create iamserviceaccount \
+--name rds-access \
+--namespace team-a \
+--cluster ${CLUSTER_NAME} \
+--attach-policy-arn ${POLICY_ARN} \
+--approve \
+-override-existing-serviceaccounts
+``` 
+Generate a DB token which can be used for password when asked
+
+```shell
+PROXY_TERGET-ENDPOINT=k get secrets aurora-cluster-secrets  -n team-a -o json | jq -r .data.proxyEndpoint | base64 --decode
+CLUSTER_USER_NAME=k get secrets aurora-cluster-secrets  -n team-a -o json | jq -r .data.clusterUsername | base64 --decode
+
+
+aws rds generate-db-auth-token \
+--hostname ${PROXY_TERGET-ENDPOINT}  \
+--port 5432 \
+--region ${REGION} \
+--username ${CLUSTER_USER_NAME}
+```
+create the pod with psql client:
+
+```shell
+k apply -f psql-client-pod.yaml
+```
+
+Exec into the pod
+
+```shell
+k exec -it postgres-client -n team-a  -- sh
+```
+
+create a DB connection and provide the token which you generate earlier for password.
+
+```shell
+psql -h ${PROXY_TERGET-ENDPOINT}  -U ${CLUSTER_USER_NAME} -d aurorapgsqldb -W
+```
+You should be able to connect to the db now then, lets check the list of default table 
+
+```shell
+\dt
+```
+Now we can do all the required database operation.
+Note : we are providing the name of the database in the claim as:  aurorapgsqldb
+
+### Deleting the resources:
+
+First delete the psql client 
+```shell
+k delete -f psql-client-pod.yaml
+```
+Delete the PostgreSQL Database by deleting the claim
+```shell
+k delete -f aurora-postgresql.yaml
+```
+Note: It will take around ~15-20 min delete the whole cluster
+
+Delete the composition and XRDs
+```shell
+cd ../composition/upbound-aws-provider/aurora
+k delete -f aurora.yaml
+k delete -f defination.yaml
+```
+Note : The role aurora-monitoring & rds-proxy will eventually be part of composition , till then it will be required to create these roles external to the composition.
```

Comment:
Move to top and include github issue.

Reply:
Created an issue and provided the reference. The comment is now placed on top.